deploy/chart/templates/0000_50_olm_07-olm-operator.deployment.yaml

```
@@ -27,7 +27,7 @@ spec:
           - /bin/olm
           args:
           - -namespace
-          - ${OPERATOR_NAMESPACE}
+          - $(OPERATOR_NAMESPACE)
```

```suggestion
          - "$(OPERATOR_NAMESPACE)"
```

The [1.14 docs](https://v1-14.docs.kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config) wrap the expansion with quotes. Not sure if it makes a difference though - waiting on CI.